src/main/jbake/content/blog/2016/05/02/hawkular-metrics-0.15.0.Final-released.md

```diff
@@ -0,0 +1,91 @@
+
+
+Here is a list of major changes:
+
+1. **Cassandra 3.5**
+    - Cassandra 3.5 is now the supported version of Cassandra
+    - Cassandra 2.2.x support is deprecated
+1. **Schema Management Tools**
+    - First release with tooling for schema change management
+    - Only upgrades are supported; incremental schema changes are installed when Hawkular Metrics starts
+    - Going forward new versions of Hawkular Metrics can be installed without the need to start with a fresh database or manually update existing database even if the schema has been modified in the new version
+    - For more details: [HWKMETRICS-361](https://issues.jboss.org/browse/HWKMETRICS-361), [Cassalog](https://github.com/jsanda/cassalog)
+1. **`*/stats` & `*/raw` Replace `*/data`**
+    - `*/data` has been **deprecated* and functionality split into two single purpose endpoints, this is applicable for all metric types (gauge, counter, availability, and string)
+    - `*/stats` endpoints return bucketed, statistical or query-time aggregated data
+    - `*/raw` endpoints accept and return raw data for a metric
+    - `*/data` is being deprecated, will not receive any updates going forward, and will be removed in future releases. There is no clear timeline for the removal since a lot of clients use it; it will be around at least for another two releases.
+    - Please update your code to use the new endpoints and follow the release notes for more details regarding removal.
+    - For more details: [HWKMETRICS-24](https://issues.jboss.org/browse/HWKMETRICS-24), [HWKMETRICS-57](https://issues.jboss.org/browse/HWKMETRICS-57)
+1. **Data Point Tags**
+    - An optional set of tags can be supplied with each data point when inserting data. Unlike metric definition tags, data point tags cannot be modified. [HWKMETRICS-368](https://issues.jboss.org/browse/HWKMETRICS-368), [HWKMETRICS-54](https://issues.jboss.org/browse/HWKMETRICS-54)
+    - Tag based bucketing for data points is supported for counters and gauges [HWKMETRICS-377](https://issues.jboss.org/browse/HWKMETRICS-377), [HWKMETRICS-373](https://issues.jboss.org/browse/HWKMETRICS-373)
+    - New endpoints for filtering and grouping data by tags:
+      - `GET /hawkular/metrics/gauges/{id}/stats/tags/{tags}`
+      - `GET /hawkular/metrics/counters/{id}/stats/tags/{tags}`
+1. **Tags**
+    - Deleting tags only requires the tag keys and not the values, this simplifies the process for tag deletion. For backwards compatibility, the API will still accept name value pairs but will not take the value into account [HWKMETRICS-385](https://issues.jboss.org/browse/HWKMETRICS-385)
+    - A new endpoint was added to query for the available values contained in a tag [HWKMETRICS-197](https://issues.jboss.org/browse/HWKMETRICS-197)
+      - Endpoint: `GET hawkular/metrics/gauge/tags/{tags}`
+      - Example: `GET hawkular/metrics/gauge/tags/hostname:*01` returns `hostname: ["web01prod", "web01qa", "backend01prod", "backend01qa" ]`
```

Comment:
My bug in the original ticket, but this query should be ``hostname:*01*`` to match those tags (or then the returned values should be changed)